lib/agentx.c

```diff
@@ -83,7 +88,10 @@ static void agentx_timeout(struct event *t)
 		snmp_timeout();
 		run_alarms();
 	} else {
-		zlog_err("%s mutex already locked", __func__);
+		if (nb_locktry_timeout_fail % 100 == 0)
```

Comment:
Could you bundle this fixup with e464755a1e26091fe9c9da74947a9ddc6d067c74? Also, what should an operator do if I see such an error?

Reply:
taken into account.